kernel/dev/checkstyle.xml

```diff
@@ -198,7 +201,8 @@
         <module name="ImportOrder">
             <property name="separated" value="true"/>
             <property name="ordered" value="true"/>
-            <property name="groups" value="java.,javax.,scala,*,io.delta.kernel,io.delta.kernel.internal"/>
```

Comment:
Given the `default` namespace going to be `io.delta.kernel.default.internal` for default internal, do you want to add it to the order?

Reply:
once we make that change how do we want our imports ordered in the default module? do we want the "default" module packages grouped separately?

<java, scala>

<3rd party>

<io.delta.kernel>
<io.delta.kernel.internal>

<io.delta.kernel.default>
<io.delta.kernel.default.internal>

Reply:
No strong preference, my main question was to not mix the `io.delta.kernel.default.internal` together with `io.delta.kernel`.

Also like the above order.